src/tryceratops/files/discovery.py

```diff
@@ -10,7 +10,7 @@
 
 from .parser import parse_file
 
-logger = logging.getLogger(__name__)
+logger = logging.getLogger(__package__)
```

Comment:
I'm curious, why `__package__`?

Reply:
That seemed to get the logger that was set to DEBUG from __main__.py when the -v flag was passed. Is there a better / different way to get it?

Reply:
If I add a new logger and put logger = logging.getLogger("tryceratops") I think that will take care of the issue ... same as above